### PR TITLE
Webinspektor - new parser

### PR DESCRIPTION
**Pull request by dargen3**
There is new parser for parse data from https://app.webinspector.com/public/recent_detections/.

### CODE REVIEW

File: intelmq/bots/parsers/webinspektor/parser.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            if parser.lsCSP:
+                index += 1
+                parser.feed(report_list[index])
+                event = self.new_event(report)
+                raw_url_line = report_list[index]
+                event.add("source.url", parser.lsData)
+                event.add("classification.type", "blacklist")
+                index += 1
+                parser.feed(report_list[index])
+                event.add("classification.taxonomy", parser.lsData)
```
Comment (wagner-certat) [inline intelmq/bots/parsers/webinspektor/parser.py]: The taxonomy is a fixed list, see https://www.enisa.europa.eu/topics/csirt-cert-services/community-projects/existing-taxonomies and our docs at https://github.com/certtools/intelmq/blob/develop/docs/Data-Harmonization.md#classification

You could save this data in the identifier. The (currently it's only one, but in theory there could be more) valid taxonomy for type `blacklist` is `Other`.

**Comment by wagner-certat** [comment]
Damned, merged that by accident. Could you please have a look at my comments? Thanks.

**Comment by dargen3** [comment]
After adjustment should i make new pull request?

**Comment by wagner-certat** [comment]
No, pull in the changes from the branch directly. Thanks!

**Pull request merged**